server-protocol.md

```diff
@@ -226,32 +234,23 @@ coming from the other client.
 Each mailbox is "opened" by some number of clients at a time, until all
 clients have closed it. Mailboxes are kept alive by either an open client, or
 a Nameplate which points to the mailbox (so when a Nameplate is deleted from
-inactivity, the corresponding Mailbox will be too).
+inactivity, the corresponding Mailbox will be too). The mailbox server makes
+sure a mailbox can never be opened by more than two parties, neither
+simultaneously not cumulatively. Trying to open a mailbox as a third party
+results in a "crowded" error for that attempt.
 
 The `open` command both marks the mailbox as being opened by the bound side,
 and also adds the WebSocket as subscribed to that mailbox, so new messages
 are delivered immediately to the connected client. There is no explicit ack
 to the `open` command, but since all clients add a message to the mailbox as
 soon as they connect, there will always be a `message` response shortly after
 the `open` goes through. The `close` command provokes a `closed` response.
-
-The `close` command accepts an optional "mood" string: this allows clients to
-tell the server (in general terms) about their experiences with the wormhole
-interaction. The server records the mood in its "usage" record, so the server
-operator can get a sense of how many connections are succeeding and failing.
-The moods currently recognized by the Rendezvous Server are:
-
-* `happy` (default): the PAKE key-establishment worked, and the client saw at
-  least one valid encrypted message from its peer
-* `lonely`: the client gave up without hearing anything from its peer
-* `scary`: the client saw an invalid encrypted message from its peer,
-  indicating that either the wormhole code was typed in wrong, or an attacker
-  tried (and failed) to guess the code
-* `errory`: the client encountered some other error: protocol problem or
-  internal error
-
-The server will also record `pruney` if it deleted the mailbox due to
-inactivity, or `crowded` if more than two sides tried to access the mailbox.
+Note that the `claim` and `allocate` commands implicitly already open the
+mailbox. The `open` command may be issued nevertheless for client implementation
+simplicity, and then simply be a no-op. There is also the possibility to open
```

Comment:
It's required, because this is how the client learns the `mailbox_id`.

Reply:
Yeah I've thought of that too. Either a client doesn't, which technically works fine until it has to reconnect … or we add the `mailbox` to the `claimed` response, or we sent an `opened` message as second answer to the `claim` command as if there had been an `open`. I'm unsure about backwards compatibility in the third case, but the second one would definitely work.